src/libnvme-mi.map

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Probably a good idea would be to add the symbol version from day 1 on. But this is something we can do after a merge.

Reply:
OK, happy to take your guidance on this.